packages/core/src/components/badge/badge.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, Host, Prop, h } from '@stencil/core'
+
+@Component({
+  tag: 'atom-badge',
+  styleUrl: 'badge.scss',
+  shadow: true,
+})
+export class AtomBadge {
+  @Prop() type:
+    | 'primary'
+    | 'secondary'
+    | 'tertiary'
+    | 'success'
+    | 'warning'
+    | 'danger'
+    | 'light'
+    | 'dark' = 'primary'
+
+  render() {
+    return (
+      <Host>
```

Comment:
**suggestion** here as we use ion-badge directy without using any wrapping style on the host, if i am not wrong the <host> may be unecessary and can be removed.

Reply:
Nice suggestion!!

Made: https://github.com/juntossomosmais/atomium/pull/239/commits/e45c65da6524c6f031bfcf7ee0ad8dd3ce8f3212